README.md

```diff
@@ -873,7 +873,9 @@ TODO-B: - no incautious use of malloc/sprintf (the language has no direct MM)
 TODO: - no use of sudo, gksu, pkexec, or LD_LIBRARY_PATH (usage is OK inside
 TODO:   tests)
 TODO: - no use of user nobody
+TODO:   (consider at least `grep` for it and `find . -user nobody`)
 TODO: - no use of setuid
+TODO:   (consider at least `grep` for it and `find . -perm -4000`)
```

Comment:
The find needs to be run on the binary package and I think that it’s not clear enough to the user. I personally limits it to grep and while checking `debian/rules`, plus looking at a package build, like https://launchpadlibrarian.net/661888717/buildlog_ubuntu-lunar-amd64.unity_7.7.0+23.04.20230222.2-0ubuntu2_BUILDING.txt.gz, where you have at the bottom the listing of the files (to check the FHS) and also the permissions + users.

Would maybe mentionning this explicitely be a better alternative?

Reply:
Added "where to search"